embark/manage.py

```diff
@@ -2,11 +2,23 @@
 """Django's command-line utility for administrative tasks."""
 import os
 import sys
+import logging
 
 
 def main():
     """Run administrative tasks."""
     os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'embark.settings')
+
```

Comment:
you could explain what the code is doing in the comments

Reply:
beautifying the log message following the scheme:
DATE - TIME - USER - LOG LEVEL - Message